rust/pact_matching/src/query.rs

```diff
@@ -13,22 +13,24 @@ use crate::matchingrules::compare_lists_with_matchingrules;
 
 /// Match the query parameters as Maps
 pub(crate) fn match_query_maps(
-  expected: HashMap<String, Vec<String>>,
-  actual: HashMap<String, Vec<String>>,
+  expected: HashMap<String, Vec<Option<String>>>,
+  actual: HashMap<String, Vec<Option<String>>>,
   context: &dyn MatchingContext
 ) -> HashMap<String, Vec<Mismatch>> {
   let mut result: HashMap<String, Vec<Mismatch>> = hashmap!{};
   for (key, value) in &expected {
+    let expected_value = value.iter().map(|v| v.clone().unwrap_or_default()).collect_vec();
```

Comment:
And here again

Reply:
Just for display purposes, keeps previous behaviour